demos/text-field.html

```diff
@@ -231,11 +230,21 @@ <h2>Text Field box</h2>
           <input id="box-required" type="checkbox">
           <label for="box-required">Required</label>
         </div>
+        <div>
+          <input id="box-use-helper-text" type="checkbox">
+          <label for="box-use-helper-text">Use Helper Text</label>
+        </div>
+        <div>
+          <input id="box-persistent-helper-text" type="checkbox" disabled>
+          <label for="box-persistent-helper-text">Make helper text persistent</label>
+        </div>
         <script>
           demoReady(function() {
+            var section = document.getElementById('text-field-box-example');
             var tfEl = document.getElementById('tf-box-example');
             var tf = new mdc.textField.MDCTextField(tfEl);
             var wrapper = document.getElementById('demo-tf-box-wrapper');
+            var helperText = section.querySelector('.mdc-text-field-helper-text');
```

Comment:
Let's give this element an `id` attribute and use `getElementById` instead

Reply:
Also will remove line 243...no longer using section